CHANGES

```diff
@@ -1,6 +1,10 @@
 Version 8.5 (Unreleased)
 ------------------------
 
+- Breadcrumb interface now rendered in event UI
```

Comment:
I noticed that most of the CHANGES entries don't have links to PRs, blog posts, etc.

Given that this is for on premise-users, I'm not sure we need spell out everything.